### PR TITLE
[MNOE-1164] Change order summary and confirm ui

### PR DESCRIPTION
**Pull request by iseessel**
Moved pricing tables around
Switched orderType to show always (not just for applications).
Added second table to summary page to be consistent with confirm page.

### CODE REVIEW

File: src/app/views/provisioning/confirm.html [hunk 1]
```diff
@@ -9,55 +9,55 @@
...
     <mno-widget-body class="no-padding">
-      <table class="table table-striped" ng-show="(vm.singleBilling && vm.billedLocally) || vm.subscription.product.product_type != 'application'">
+      <table class="table table-striped">
         <thead>
         <tr>
-          <th>{{ 'mnoe_admin_panel.dashboard.provisioning.confirm.name' | translate }}</th>
-          <th>{{ 'mnoe_admin_panel.dashboard.provisioning.confirm.order_type' | translate }}</th>
-          <th>{{ 'mnoe_admin_panel.dashboard.provisioning.confirm.description' | translate }}</th>
-          <th style="width: 25%">{{ 'mnoe_admin_panel.dashboard.provisioning.confirm.price' | translate }}</th>
+          <th style="width: 33%">{{ 'mnoe_admin_panel.dashboard.provisioning.confirm.order_type' | translate }}</th>
```
Comment (adamaziz15) [inline src/app/views/provisioning/confirm.html]: Can we put the styling in `confirm.less`?

File: src/app/views/provisioning/confirm.html [hunk 2]
```diff
@@ -9,55 +9,55 @@
...
       </table>
 
-      <table class="table table-striped" ng-show="vm.subscription.product.product_type == 'application'">
+      <table class="table table-striped" ng-show="(vm.singleBilling && vm.billedLocally) || vm.subscription.product.product_type != 'application'">
         <thead>
         <tr>
-          <th>{{ 'mnoe_admin_panel.dashboard.provisioning.confirm.pricing_info.title' | translate }}</th>
+          <th>{{ 'mnoe_admin_panel.dashboard.provisioning.confirm.name' | translate }}</th>
+          <th>{{ 'mnoe_admin_panel.dashboard.provisioning.confirm.description' | translate }}</th>
+          <th style="width: 25%">{{ 'mnoe_admin_panel.dashboard.provisioning.confirm.price' | translate }}</th>
```
Comment (adamaziz15) [inline src/app/views/provisioning/confirm.html]: confirm.less please :)

File: src/app/views/provisioning/summary.html [hunk 3]
```diff
@@ -9,19 +9,42 @@
       </div>
     </mno-widget-header>
     <mno-widget-body class="no-padding">
+      <table class="table table-striped">
+        <thead>
+        <tr>
+          <th style="width: 33%">{{ 'mnoe_admin_panel.dashboard.provisioning.summary.order_type' | translate }}</th>
```
Comment (adamaziz15) [inline src/app/views/provisioning/summary.html]: summary.less

File: src/app/views/provisioning/summary.html [hunk 4]
```diff
@@ -9,19 +9,42 @@
...
+        </tbody>
+      </table>
+
       <table class="table table-striped" ng-show="vm.subscription.product_pricing">
         <thead>
         <tr>
           <th>{{ 'mnoe_admin_panel.dashboard.provisioning.summary.name' | translate }}</th>
-          <th>{{ 'mnoe_admin_panel.dashboard.provisioning.summary.order_type' | translate }}</th>
           <th>{{ 'mnoe_admin_panel.dashboard.provisioning.summary.description' | translate }}</th>
           <th style="width: 25%">{{ 'mnoe_admin_panel.dashboard.provisioning.summary.price' | translate }}</th>
```
Comment (adamaziz15) [inline src/app/views/provisioning/summary.html]: I know this one wasn't added by you but => summary.less please :)

**Comment by iseessel** [comment]
@adamaziz15 Less variables updated.

File: src/app/views/provisioning/summary.html [hunk 5]
```diff
@@ -9,19 +9,42 @@
...
```
Comment (adamaziz15) [inline src/app/views/provisioning/summary.html]: Missed this one ^

File: src/app/views/provisioning/summary.html [hunk 6]
```diff
@@ -9,19 +9,42 @@
...
+          <th id="order-type">{{ 'mnoe_admin_panel.dashboard.provisioning.summary.order_type' | translate }}</th>
+          <th ng-show="vm.subscription.product.product_type == 'application'">{{ 'mnoe_admin_panel.dashboard.provisioning.summary.pricing_info.title' | translate }}</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+          <td>{{ vm.orderTypeText | translate }}</td>
+          <td ng-show="vm.subscription.product.product_type == 'application'">
+            <span ng-hide="vm.singleBilling">
+              {{ 'mnoe_admin_panel.dashboard.provisioning.summary.pricing_info.single_billing_disabled' | translate }}
```
Comment (ouranos) [inline src/app/views/provisioning/summary.html]: Rather than swapping between span, would it be easier to have the controller returning the locale key?

Comment (iseessel) [reply src/app/views/provisioning/summary.html]: sure

**Pull request merged**